LoopKit/GlucoseKit/GlucoseMath.swift

```diff
@@ -48,7 +48,7 @@ extension BidirectionalCollection where Element: GlucoseSampleValue, Index == In
 
     /// Whether the collection contains no calibration entries
     /// Runtime: O(n)
-    var isCalibrated: Bool {
+    public func isCalibrated() -> Bool {
```

Comment:
maybe `containsCalibration`? I wouldn't say the trace "is calibrated", just because it contains an entry that is a calibration.

Reply:
And why is it turning into a function?  It takes no args and has no side effects, so it could still remain a var?

Reply:
Agreed on the naming! I made it a function since I normally think of variables as having constant-time access; happy to stick to a variable since it's a change that's not related to the core of the PR

Reply:
I'm honestly a little confused by the comment ("contains no calibration entries") versus the name ("isCalibrated"). It seems like this function is seeing if we have no points shifted for visual consistency after calibration, so I would expect the name to be something like "isNotCalibrated". Which interpretation would be correct?

Reply:
Yeah, the function name is not great; probably `containsCalibrations()` is better.

Reply:
Or negated, `containsNoCalibrations()`